specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/examples/Skus_List.json

```diff
@@ -0,0 +1,86 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-10-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "resourceType": "caches",
+            "name": "Standard_2G",
+            "locations": [
+              "East US"
+            ],
+            "locationInfo": [
+              {
+                "location": "East US",
```

Comment:
location does not have spaces I think.. 
minor change - "eastus"

Reply:
Fixed in next commit.

Reply:
Fixed in next commit.